src/forklift/engine.py

```diff
@@ -467,6 +467,17 @@ def git_update():
     return [error for error, info in results if error is not None]
 
 
+def gift_wrap(destination, source=None):
+    if exists(destination):
+        destination = join(destination, 'gift-wrapped')
```

Comment:
copytree expects an empty directory

Reply:
So if you wanted the gdbs to end up in the receiving directory you would have to manually copy them after running the tool?

Reply:
Yes.